rag_app/README.md

```diff
@@ -0,0 +1,47 @@
+# Introduction
+
+This is a README detailing how to use `rag-app`, a simple cli which can be used to interact with a local lancedb database. You can install this by running the command
+
+```
+pip3 install -e .
+```
+
+
+# Commands
+
+## Querying the Database
+
+We can use the command `rag-app query db <db path> <db table> <query>` in order to query our database. We use the `text-embedding-3` model with a dimensionality of 256 by default.
+
+```
+>> rag-app query db ./db paul_graham "What's the biggest challenge facing any startup"                                                             
+┏━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃            ┃                                                                                                                            ┃
+┃  Chunk ID  ┃  Result                                                                                                                    ┃
```

Comment:
Lets add more metadata to the chunks like 

1. title of post
2. chunk i out of n for that post 
3. publish date of the post.

Reply:
Sounds good to me, let me figure out how to support Gray matter with python